Pandoc dependency when installing.

The Makefile depends on Pandoc now.

```
$ docker run -it --rm erlang:18 bash -c 'cd /usr/local/lib/erlang/lib && git clone https://github.com/rvirding/lfe.git && cd lfe && git checkout v1.0.2 && make compile install'
...
bin/lfe bin/lfec -I include -o ebin -pa ../lfe src/cl.lfe
ecp: #(cl ())
cp src/lfe.app.src ebin/lfe.app
rm src/lfe_scan.erl
make[1]: Leaving directory '/usr/local/lib/erlang/lib/lfe'
pandoc -f markdown -s -t man -o doc/man/lfe.1 doc/src/lfe.1.md
make: pandoc: Command not found
Makefile:159: recipe for target 'doc/man/lfe.1' failed
make: *** [doc/man/lfe.1] Error 127
```

--- a/Makefile
+++ b/Makefile
@@ -238,7 +238,7 @@ $(EPUBDIR)/%.epub: $(DOCSRC)/%.3.md
 $(EPUBDIR)/%.epub: $(DOCSRC)/%.7.md
 	pandoc -f markdown -t epub -o $@ $<
 
-install-man: docs-man
+install-man:
 	@mkdir -p $(MANINSTDIR)/man1 $(MANINSTDIR)/man3 $(MANINSTDIR)/man7
 	cp $(MANDIR)/*.1 $(MANINSTDIR)/man1/
 	cp $(MANDIR)/*.3 $(MANINSTDIR)/man3/

